src/MoBi.Core/Domain/Model/MoBiSimulation.cs

```diff
@@ -122,6 +133,24 @@ public IReadOnlyList<IBuildingBlock> BuildingBlocks()
          return buildingBlocks;
       }
 
+      public IReadOnlyList<ParameterValue> OriginalQuantityValues => _quantityValueCache.ToList();
+
+      public void AddOriginalQuantityValue(ParameterValue parameterValue)
+      {
+         if (_quantityValueCache[parameterValue.Path] == null)
+            _quantityValueCache[parameterValue.Path] = parameterValue;
+      }
+
+      public void RemoveOriginalQuantityValue(ObjectPath objectPath)
+      {
+         _quantityValueCache.Remove(objectPath);
+      }
+
+      public ParameterValue OriginalQuantityValueFor(ObjectPath objectPath)
+      {
```

Comment:
what about you inline some the calll in this  file as we have a bunch of => already so that it's consistent